wazo_load_api/plugins/job/http.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from fastapi import APIRouter
+from .services import run
+
+router = APIRouter()
+
+
+@router.post("/run")
+async def run_load(payload: dict):
+    try:
+        command = payload["cmd"]
+    except KeyError as e:
+        return {"status": "internal_error", "error": str(e)}
```

Comment:
I think the proper way to return an error with fastapi is to raise an HTTPException
```python
from fastapi import HTTPException
raise HTTPException(status_code=400, detail={'status': 'internal_error', 'error': str(e)})
```